airwallex/src/main/java/com/airwallex/android/view/util/AvailablePaymentMethodExtensions.kt

```diff
@@ -1,8 +1,21 @@
 package com.airwallex.android.view.util
```

Comment:
Could the naming of extension file align with the existing pattern `AvailablePaymentMethod+Extensions` or `AvailablePaymentMethodUtils`?

Reply:
ok